Source/FieldSolver/SpectralSolver/SpectralAlgorithms/GalileanPsatdAlgorithmRZ.cpp

```diff
@@ -0,0 +1,356 @@
+/* Copyright 2019-2020 David Grote
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#include "WarpX.H"
+#include "GalileanPsatdAlgorithmRZ.H"
+#include "Utils/WarpXConst.H"
+#include "Utils/WarpXProfilerWrapper.H"
+
+#include <cmath>
+
+using namespace amrex::literals;
+
+
+/* \brief Initialize coefficients for the update equation */
+GalileanPsatdAlgorithmRZ::GalileanPsatdAlgorithmRZ (SpectralKSpaceRZ const & spectral_kspace,
+                                                    amrex::DistributionMapping const & dm,
+                                                    int const n_rz_azimuthal_modes, int const norder_z,
+                                                    bool const nodal,
+                                                    const amrex::Array<amrex::Real,3>& v_galilean,
+                                                    amrex::Real const dt)
+     // Initialize members of base class
+     : SpectralBaseAlgorithmRZ(spectral_kspace, dm,
+                               norder_z, nodal),
+       m_dt(dt)
+{
+    m_v_galilean = v_galilean;
```

Comment:
Would it be possible to use member initialization for `m_v_galilean` as well?
```suggestion
     : SpectralBaseAlgorithmRZ(spectral_kspace, dm, norder_z, nodal),
       m_dt(dt),
       m_v_galilean(v_galilean)
{
```